tools/js-test-runner/jest-config/jest.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+let config = {};
+
+const defaultConfig = {
+	preset: '@wordpress/jest-preset-default',
+	rootDir: process.cwd(),
+	testEnvironment: 'jsdom',
+	// TODO: remove as soon gutenberg release new version
+	//https://github.com/WordPress/gutenberg/blob/%40wordpress/jest-preset-default%407.1.5-next.33ec3857e2.0/packages/jest-preset-default/jest-preset.js
+	testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/test/*.[jt]s?(x)', '**/?(*.)test.[jt]s?(x)' ],
+	moduleNameMapper: {
+		'\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': path.join(
+			__dirname,
+			'./fileMock.js'
+		),
+		'\\.module\\.(css|less)$': 'identity-obj-proxy',
+		jetpackConfig: path.join( __dirname, './jetpack.config' ),
+	},
+	coverageReporters: [ 'clover', 'lcov' ],
+	collectCoverageFrom: [
+		'**/*.{js,jsx}',
+		'!**/node_modules/**',
+		'!**/vendor/**',
+		'!**/__tests__/**',
+		'!**/coverage/**',
+		'!test-main.{js,jsx}',
+		'!jest.setup.{js,jsx}',
+	],
+	transform: {
+		'\\.[jt]sx?$': [
+			require.resolve( 'babel-jest' ),
+			{ configFile: path.join( __dirname, './babel.config.js' ) },
+		],
+	},
+};
```

Comment:
If someone just wants to add to `collectCoverageFrom` or `moduleNameMapper`, they're going to have to duplicate all the existing entries instead of being able to extend them. Maybe put the `defaultConfig` in a separate file, so the config could `require( 'jetpack-js-test-runner/jest-config/defaultConfig.js' )` to do something like that?

Reply:
@anomiex I'm already doing the merge into lines 45 and 46, not? I require the custom config and merge with the default config, making sure that the custom config takes priority.

Reply:
Say someone wants to add `**/*.mjs` into `collectCoverageFrom`. As it is, if they specify
```
collectCoverageFrom: [ '**/*.mjs' ]
```
in their custom config, then they'll lose all the other entries from lines 25–31. If they want to keep them, they'd have to copy-paste them all into their custom config.

Reply:
@anomiex oh! gotcha ... I'll use `merge` from `lodash` to make sure that we merge the values, do you have any other way to do that?

Reply:
I note using `merge` might make it difficult to remove an entry from `collectCoverageFrom`, if for some reason someone wanted to do that. That's why I suggested giving the ability for the custom config to load the default config, so then it can do whatever it wants to munge it.

Reply:
@anomiex makes sense! will follow your suggestion